test/test_nestedtensor.py

```diff
@@ -3044,6 +3044,18 @@ def test_unbind(self, device):
             for i, t in enumerate(out):
                 self.assertEqual(t, tensor_list[i])
 
+    def test_embedding(self, device):
```

Comment:
There is already a test for this https://github.com/pytorch/pytorch/blob/main/test/test_nestedtensor.py#L795 but for the other layout. Can we unify these two?

Reply:
I wrote the original test too... lol. But any suggestions on how to better unify the tests? I guess one option is adding an extra parameter to the tests for the layout in a new class or something, but I saw some mention of OpInfo in the comments of the file (which I don't know anything about)

Reply:
Haha, that's right! Full circle.

Eventually we can migrate some of these tests to opinfos (we can't reimplement all tests for NT as well).

I think adding an extra parameter for layout is a good start at least in context of this PR.

Reply:
It's ok to do this in a follow up PR as well. We can merge this one for now as is.